assets/src/stories-editor/index.js

```diff
@@ -216,18 +214,6 @@ subscribe( async () => {
 	}
 
 	allBlocksWithChildren = getClientIdsWithDescendants();
-
-	// Re-add controls when switching back from code to visual editor.
-	const newEditorMode = getEditorMode();
-	if ( 'visual' === newEditorMode && newEditorMode !== editorMode ) {
-		while ( ! document.querySelector( '.editor-block-list__layout' ) ) {
```

Comment:
This is the worst loop I have ever seen 😱.

Reply:
I'm not proud of it 😅